modules/mergesort/include/merge_sort.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file merge_sort.h
+ * @author WARG
+ *
+ * @section LICENSE
+ *
+ *  Copyright (c) 2015, Waterloo Aerial Robotics Group (WARG)
+ *  All rights reserved.
+ *
+ *  inthis software is licensed under a modified version of the BSD 3 clause license
```

Comment:
Didn't see this before. Just a typo, and given this is the bootcamp you can ignore this now.